spec/features/bulk_updates/bulk_updates_science_keywords_spec.rb

```diff
@@ -390,11 +397,13 @@
         end
 
         it 'displays the updated keywords' do
-          within '.science-keywords-preview' do
+          within '#metadata-preview' do
             expect(page).to have_no_content('EARTH SCIENCE ATMOSPHERIC TEMPERATURE SURFACE TEMPERATURE', normalize_ws: true)
```

Comment:
These need to be split up into multiple expects, one for each keyword.

Reply:
Again for this you probably don't want to have multiple expects, just `expect(page).to have_no_content('SURFACE TEMPERATURE', normalize_ws: true)`